experiments/PlaywrightSharp/PlaywrightSharp/ApiSurface/IPage.cs

```diff
@@ -67,7 +67,7 @@ public interface IPage
 		event EventHandler<IFrame> FrameDetached;
 		event EventHandler<IFrame> FrameNavigated;
 		event EventHandler<IPage> OnLoad;
-		event EventHandler<Error> PageError;
+		event EventHandler<Exception> PageError;
```

Comment:
I don't if we should use an Exception to return page error data.

Reply:
It's `event EventHandler<string> PageError;` for now, but we might consider introducing some special object later down the line, if we feel like we want to.